chromium_src/chrome/browser/ui/webui/settings/browser_lifetime_handler.cc

```diff
@@ -20,7 +21,7 @@ namespace settings {
 BrowserLifetimeHandler::~BrowserLifetimeHandler() {}
 
 void BrowserLifetimeHandler::HandleRelaunch(const base::Value::List& args) {
-#if BUILDFLAG(ENABLE_SPARKLE)
+#if BUILDFLAG(ENABLE_SPARKLE) && !BUILDFLAG(BRAVE_ENABLE_UPDATER)
```

Comment:
This change makes the "Relaunch" button on `brave://settings/help` act as expected, instead of being a no-op, when `brave_enable_updater` is true.